src/components/NcPasswordField/NcPasswordField.vue

```diff
@@ -127,7 +127,7 @@ import { generateOcsUrl } from '@nextcloud/router'
 import { t } from '../../l10n.js'
 import logger from '../../utils/logger.js'
 
-const defaultPasswordPolicy = loadState('core', 'capabilities', { passwordPolicy: null }).password_policy
+const defaultPasswordPolicy = loadState('core', 'capabilities', { password_policy: null })?.password_policy || null
```

Comment:
```suggestion
const defaultPasswordPolicy = loadState('core', 'capabilities', {}).password_policy || null
```

Should even be enough I think

Reply:
The question mark was needed for talks test, but maybe I messed up the compiling order between vue and talk when testing. But can retry tomorrow

Reply:
Seems you are right. the `|| null` basically seems to kill the error already.

Reply:
It would fail if someone provided an initial state that is not an object. But I don't see why that would happen anyway :)